docs/running-rules.md

```diff
@@ -9,49 +9,49 @@ import MoreHelp from "/src/components/MoreHelp"
 
 ## Introduction
 
-Rules are specific patterns based on which Semgrep reports findings in code. These findings may help you to catch issues of security, performance, correctness, and other bugs. Rules are stored in an open-source [Semgrep Registry](https://github.com/returntocorp/semgrep-rules) that enables you to scan code without need to write anything custom. Use Semgrep rules in the following ways:
+Semgrep reports findings in code based on rules. These rules use specific patterns to match code. When Semgrep matches a portion of code, it is reported as a finding. Findings may help you to catch issues of security, performance, correctness, and best practices. Rules are stored in an open-source [Semgrep Registry](https://github.com/returntocorp/semgrep-rules) that enables you to scan code without need to write anything custom. The list below covers different kinds of Semgrep rules:
```

Comment:
The way you had this previously started with what rules are in the first sentence, and now we wait until the second sentence. Did I miss a comment that suggested this part should be rearranged? I think it's clearer to define what rules are up-front

Reply:
@efortuna ok! Please, feel free to suggest an edit here.

Reply:
@efortuna So Sara suggested the original [thing here](https://github.com/returntocorp/semgrep-docs/pull/551#discussion_r901233736) - so the current state of this paragraph is:

```markdown
Semgrep reports findings in code based on rules, which use specific patterns to match code. When Semgrep matches a portion of code, it is reported as a finding. Findings may help you to catch issues of security, performance, correctness, and best practices. Rules are stored in an open-source [Semgrep Registry](https://github.com/returntocorp/semgrep-rules) that enables you to scan code without the need to write anything custom. The list below covers different kinds of Semgrep rules:
```

Reply:
just to follow up on this comment, I made a suggestion in the latest version.